### ISSUE TITLE
Helioviewer website is not accessible 2021-JUNE-26

### ISSUE THREAD

**Issue by DawnMarieMueller** [Helioviewer-Project/JHelioviewer-SWHV#169]
Hello.  I, and others in the U.S.A., are unable to access the Helioviewer website.  Error msg:  "This page isn't working right now."

I looked up Heliophysics.org website on DomainTools.com WHOIS search engine.  The site appears registered to NASA/GreenbeltMD.  I am unable to reach anyone at NASA to find out why the website is not accessible.

     https://whois.domaintools.com/helioviewer.org

AR12835 is rotated into view & needs to be monitored for flares, so help in resolving this issue would be appreciated by many.

I may be contacted by cellphone at 00 1 (779) 713-3893 and Twitter @Ancestralist and email dawnmm@mc.net for additional info.

**Comment by ebuchlin** [comment]
Hello,
https://helioviewer.ias.u-psud.fr/ is accessible. The latest AIA 17.1nm data we have right now is from 2021-06-27T06:16:22.
However, please note that these data are mirrored from helioviewer.org, so if the whole webserver at helioviewer.org is down (and not just the main website with the helioviewer app), we won't get the latest data. That doesn't seem to be the case today, so we have recent data.

**Comment by simplio** [comment]
> Hello,
> https://helioviewer.ias.u-psud.fr/ is accessible. The latest AIA 17.1nm data we have right now is from 2021-06-27T06:16:22.
> However, please note that these data are mirrored from helioviewer.org, so if the whole webserver at helioviewer.org is down (and not just the main website with the helioviewer app), we won't get the latest data. That doesn't seem to be the case today, so we have recent data.

Not true. Helioviewer France is reported newest data 6-25!

**Comment by ebuchlin** [comment]
Sorry I looked the wrong field. But 2021-06-25 is also the latest date available on the directories of helioviewer.org where we mirror from, so at least these images can be seen on https://helioviewer.ias.u-psud.fr/ while the web app at https://helioviewer.org/ is down.

**Comment by simplio** [comment]
I have no clue why data is down again within a week after I reported it in another issue I closed. Seems like a good stable server wont need to go down this much. IF this service is in any way connected with NASA maybe NASA should hear my two cents. Seems like its happening more and more as SC25 progresses.

**Comment by DawnMarieMueller** [comment]
Yes, I am able to view the IAS mirror of Helioviewer out of France, but without current images; latest appear to be June 25th.

SolarMonitor.org seems to be pulling the current SDO AIA 193 images just fine, as well as the current SDO HMI images.  So there does not seem to be a more general barrier to accessing SDO data.

I *do* note that there was some sort of announcement last week that NASA was going to have some scheduled data access issues while something was worked on, but I can't remember where I saw the announcement, & I don't immediately see it on SDAC, JSOC or some other sites.  If I find it again, I will post it here.

**Comment by ebuchlin** [comment]
Hello,
I am happy to report that we are now downloading recent SDO/AIA and HMI data from LMSAL (thanks to ROB for the suggestion), and so they should be available from https://helioviewer.ias.u-psud.fr/ and from JHelioViewer by selecting the layer to be added from the "IAS" server.
